ansible/.env.j2

```diff
@@ -0,0 +1,16 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+SECRET_KEY=secret-key
+
+# Trello API configuration
+TRELLO_API_BASE_URL=https://api.trello.com
+TRELLO_API_KEY={{trello_api_key}}
+TRELLO_API_TOKEN={{trello_api_token}}
+TRELLO_BOARD_ID=65cb80423e987af6cb9a881b
+TO_DO_LIST_ID=65cb80581416858233a02af3
+DOING_LIST_ID=65cb805af5f7675d1b0c4fe2
+DONE_LIST_ID=65cb805be846f5052ce7355a
```

Comment:
Very minor - I'd do the same for this exercise but if this were a real project I'd recommend pulling these out into variables in your playbook, even if we keep the values saved inside that as a default. They're not sensitive so we don't need to hide them, but we would likely want to eventually swap them out for a new environment
(Wouldn't bother changing here, but worth noting)